### ISSUE TITLE
Content Tile - Normal - Read more link issue

### ISSUE THREAD

**Issue by rolorogan** [wmcadigital/wmn-design-system#208]
General tile "Read More" link displays outside of the tile as position:relative needs adding to wmnds-content-tile - https://wmnetwork.netlify.app/components/content-tiles/

**Comment by daylesalmon** [comment]
@rolorogan I had to update the content tiles a bit, the new code as per the page you linked above is:

```html
<div class="wmnds-content-tile wmnds-content-tile--with-one-link">
    <h2>Major Roadworks and events</h2>
    <div class="wmnds-content-tile__copy">
      <p>See all major roadworks and events. These could affect your journey, plan ahead.</p>
      <a href="#" title="link title" target="_self" class="wmnds-link wmnds-float-right">Read more</a>
    </div>
</div>
```

Notice the `wmnds-content-tile--with-one-link` class. This will add the position relative etc. to the content tile. I added this modifier class so that when multiple content tiles are next to each other in a row, the "read more" link is always at the bottom of the tile to look consistent.

I know there is a current styling issue with longer links and when the content tiles are shrunk down to mobile. I need to come back and update the styling at some point. Unless you have any suggestions?

I've also noticed that I've left in a rogue "read more" on the first content tile example on that page. I will remove that shortly.
